.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseMergedDateTimeExtractor.cs

```diff
@@ -276,6 +276,23 @@ private bool ShouldSkipFromToMerge(ExtractResult er)
             return config.FromToRegex.IsMatch(er.Text);
         }
 
+        private bool SkipRegexTasksMode(ExtractResult er)
```

Comment:
Nit: Rename accoridng to previous suggestion.

Reply:
Renamed SkipRegexTasksMode to TasksModeFilters.